com.alizadicomviewer.AlizaMS.yaml

```
@@ -0,0 +1,158 @@
+app-id: com.alizadicomviewer.AlizaMS
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+command: alizams
+#For CastXML
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm11
+rename-desktop-file: alizams.desktop
+rename-icon: alizams
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+cleanup:
+  - /include
+  - /lib/cmake
+  - '*.la'
+  - '*.a'
+  - '/man'
+
+modules:
+    #cmake-ninja does not work
+  - name: CastXML
+    buildsystem: cmake
+    config-opts:
+      - -DCLANG_LINK_CLANG_DYLIB:BOOL=ON
+      - -DLLVM_DIR=/usr/lib/sdk/llvm11/lib/cmake/llvm
+    sources:
+      - type: archive
+        url: https://github.com/CastXML/CastXML/archive/v0.4.4.tar.gz
+        sha256: e7343edf262b291a9a3d702dbfa6d660e8ed81170454eab6af10a6dbf8c8141d
+        x-checker-data:
+          type: anitya
+          project-id: 9347
+          stable-only: true
+          url-template: https://github.com/CastXML/CastXML/archive/v$version.tar.gz
+    cleanup:
+      - /lib/cmake
+
+  - name: SWIG
+    config-opts:
+      - --without-pcre
+      - --disable-ccache
+      - --without-tcl
+      - --without-tcl
+      - --without-perl5
+      - --without-octave
+      - --without-scilab
+      - --without-java
+      - --without-ruby
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/swig/files/swig/swig-4.0.2/swig-4.0.2.tar.gz
+        sha256: d53be9730d8d58a16bf0cbd1f8ac0c0c3e1090573168bfa151b01eb47fa906fc
+        x-checker-data:
+          type: anitya
+          project-id: 4919
+          stable-only: true
+          url-template: https://sourceforge.net/projects/swig/files/swig/swig-$version/swig-$version.tar.gz
+    cleanup:
+      - /share
+
+  - name: gdcm
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DGDCM_BUILD_APPLICATIONS=OFF
+      - -DGDCM_BUILD_SHARED_LIBS=ON
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DGDCM_WRAP_PYTHON=ON
+      - -DPYTHON_EXECUTABLE=/usr/bin/python3
+      - -DGDCM_INSTALL_PYTHONMODULE_DIR=/app/lib/python3.8/site-packages/
```

```suggestion
      - -DGDCM_INSTALL_PYTHONMODULE_DIR=/app/lib/python3.9/site-packages/
```

(this implies you use the -21.08 runtime as suggested above)